src/ploigos_step_runner/step_implementers/static_code_analysis/tox_lint.py

```diff
@@ -0,0 +1,67 @@
+"""`StepImplementer` for the `static-code-analysis` step using tox and pylint
```

Comment:
@cts-zeero can you add the standard block we have with the config options and result artifacts we have in the other step implemetners so users can easly look up the inputs and outputs?

Reply:
Sure thing. Do you want me to add the docs to the `ToxTest` step implementer as well?

Reply:
As an aside, I based the module docs off of the `NpmTest` step which didn't have a options doc block.

I assumed it was because the options were defined in the generic. The generic docs will be repeated for the tox test, lint, etc docs as well since these tox implementers don't really do anything opinionated (other than silence output) to maximize flexibility.

The implementer can't really assume the intent for commands run inside the tox environment, nor the resulting artifacts, since it is basically a tox executor.

Reply:
| As an aside, I based the module docs off of the NpmTest step which didn't have an options doc block.

whoops. thats a missing doc on the Npm one then.

| I assumed it was because the options were defined in the generic. The generic docs will be repeated for the tox test, 
| lint, etc docs as well since these tox implementers don't really do anything opinionated (other than silence output) to 
| maximize flexibility.

we have been trying to put the doc in both places so people dont have to go to the generic (if there is one) to look up all the valid config for a given step

| The implementer can't really assume the intent for commands run inside the tox environment, nor the resulting 
| artifacts, since it is basically a tox executor.

understood. I am assuming/thinking that down the road we may have specific artifact/evidence gatherers for specific python modules folks may use for testing/linting/etc that they can activate to gather relevant aftifacts and evidence when giving us more information about hwo they are are running the tests or linting or such.

Reply:
Perfect. I will get this updated and committed.

I have two more steps coming soon as well 👍

Reply:
wahoo. will be on the look out.

Reply:
@cts-zeero still planning on adding the updates. We are planning a release in later half of this week and I would like to include this.